### ISSUE TITLE
Cannot install vscodium on Mac with homebrew - Error: SHA256 mismatch

### ISSUE THREAD

**Issue by erredeco** [VSCodium/vscodium#1574]
**Describe the bug**
Cannot install vscodium VSCodium.x64.1.80.1.23194.dmg on Mac with homebrew and MacOS Ventura 13.4.1 (c)

**To Reproduce**

I used the following:

```
brew cleanup
brew upgrade                
brew install --cask vscodium
```

The result is:

```
==> Downloading https://github.com/VSCodium/vscodium/releases/download/1.80.1.23194/VSCodium.x64.1.80.1.23194.d
==> Downloading from https://objects.githubusercontent.com/github-production-release-asset-2e65be/144590939/795
######################################################################################################## 100.0%
Error: SHA256 mismatch
Expected: 592e0b901e74ae345500d21df2fcf93782b15bba80171df4c32fe80d02def024
Actual: f66d476ac39c78640aec98037990ee6168ea0cc2f097a5d23a2bac912af4bdd9
```

 

**Expected behavior**
I expect the software to be installed

**Desktop**
 - OS:  MacOS Ventura 13.4.1 (c)
 - Architecture X64
 - Version 1.80.1.23194
 - App Manager: Homebrew

**Comment by daiyam** [comment]
`f66d476ac39c78640aec98037990ee6168ea0cc2f097a5d23a2bac912af4bdd9 ` is correct.

I will look how to fix the recipe and take control over it.

**Comment by daiyam** [comment]
Ok, I've fixed the issue but they will have to update the recipe.

**Comment by kara-liu** [comment]
Any updates? Having this issue too.

**Comment by grimkraut** [comment]
I'm having the same issue here

**Comment by daiyam** [comment]
> This is fixed on the latest release `1.80.1.23208` on my side.

Ya, it's what I should have done when I found out about the issue. (it didn't cross my mind at that time...)
